DSMRloggerAPI.ino

```diff
@@ -44,6 +44,10 @@
 
 #include "DSMRloggerAPI.h"
 
+#ifdef USE_MINDERGAS
+  #include "MinderGas.h"
+#endif 
+
```

Comment:
Waarom bovenstaande regels niet in DSMRloggerAPI.h?

Ik wil juist al deze includes uit dit file houden zodat je altijd weet waar je moet zoeken/kijken